lib/src/internal/map_interface.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The FlutterCandies author. All rights reserved.
+// Use of this source code is governed by an Apache license that can be found
+// in the LICENSE file.
+
+/// Contains an abstract method toMap to indicate that it can be converted into a Map object
+mixin IMapMixin{
```

Comment:
Need format for other files too.

```suggestion
mixin IMapMixin {
```